t/02-client.t

```diff
@@ -69,4 +69,48 @@ subtest "js socket", sub {
     ok($gc->_get_random_js_sock());
 };
 
+subtest 'Client: "on_fail" handler is triggered on timeout' => sub {
```

Comment:
subtest fails if no `gearmand` in `$PATH`. Could you add a skip please. [Here an example](https://github.com/p-alik/perl-Gearman/blob/master/t/02-client.t#L61).

Reply:
Added a `skip_all` as suggested.